lib/cn/kvs_mblk_desc.h

```diff
@@ -1,25 +1,25 @@
 /* SPDX-License-Identifier: Apache-2.0 */
 /*
- * Copyright (C) 2015-2020 Micron Technology, Inc.  All rights reserved.
+ * Copyright (C) 2015-2022 Micron Technology, Inc.  All rights reserved.
  */
 
 #ifndef HSE_KVS_MBLK_DESC_H
 #define HSE_KVS_MBLK_DESC_H
 
-#include <hse_util/inttypes.h>
+#include <stdint.h>
 
-#include <mpool/mpool_structs.h>
+#include <hse/types.h>
 
 struct mpool_mcache_map;
 struct mpool;
 
 struct kvs_mblk_desc {
     void *                   map_base; /* base address of mcache map */
     struct mpool_mcache_map *map;      /* mcache map */
-    u32                      map_idx;  /* index of mblk in map */
+    uint32_t                 map_idx;  /* index of mblk in map */
     enum hse_mclass          mclass;   /* media class */
     struct mpool *           ds;       /* mpool dataset */
-    u64                      mb_id;    /* mblock id */
+    uint64_t                 mb_id;    /* mblock id */
```

Comment:
Unfortunate use of mb_id vs mbid:

$ gr mbid lib|wc -l
319
(hse-build-3.8) [gbecker@hilton hse] (master)
$ gr mb_id lib|wc -l
4